b2b-sample/guardio-insurance-sample-application/components/sections/settingsSection/idpSection/idpSectionComponent.js

```diff
@@ -0,0 +1,446 @@
+                    for (key in formValue) {
+                        val = formValue[key];
+
+                        if (!val) {
+                            return true;
+                        }
+                    }
+
+                    return false;
+                }
+
+                break;
+                
+            default:
+
+                return true;  
+        }
+
+        return true;
+    };
+
+    return (
+        <Container>
+
+            <SettingsTitle
+                title="Identity Providers"
+                subtitle="Manage identity providers to allow users to log in to your application through them." />
+
+            {
+                idpList
+                    ? (<FlexboxGrid
+                        style={ { height: "60vh", marginTop: "24px", width: "100%" } }
+                        justify={ idpList.length === 0 ? "center" : "start" }
+                        align={ idpList.length === 0 ? "middle" : "top" }>
+                        { idpList.length === 0
+                            ? (<EmptyIdentityProviderList
+                                onAddIdentityProviderClick={ onAddIdentityProviderClick }
+                            />)
+                            : (<IdentityProviderList
+                                fetchAllIdPs={ fetchAllIdPs }
+                                idpList={ idpList }
+                                session={ session }
+                            />)
+                        }
+                    </FlexboxGrid>)
+                    : null
+            }
+
+            {
+                openAddModal && (
+                    <AddIdentityProviderModal
+                        templates={ templates }
+                        onClose={ () => setOpenAddModal(false) }
+                        openModal={ openAddModal }
+                        onTemplateSelected={ (template) => {
+                            setOpenAddModal(false);
+                            setSelectedTemplate(template);
+                        } }
+                    />
+                )
+            }
+            {
+                selectedTemplate && (
+                    <IdPCreationModal
+                        onSave={ onIdPSave }
+                        onCancel={ onCreationDismiss }
+                        openModal={ !!selectedTemplate }
+                        template={ selectedTemplate } />
+                )
+            }
+        </Container>
+    );
+
+}
+
+/**
+ * 
+ * @param prop - openModal (function to open the modal), onClose (what will happen when modal closes), 
+ *               templates (templates list), onTemplateSelected 
+ *              (what will happen when a particular template is selected)
+ * @returns A modal to select idp's
+ */
+const AddIdentityProviderModal = (prop) => {
+
+    const { openModal, onClose, templates, onTemplateSelected } = prop;
+
+    const resolveIconName = (template) => {
+        if (GOOGLE_ID === template.templateId) {
+
```

Comment:
Shall we check that this new line is required after if statement? Check other places as well.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/130d957728499e5e0e731eba0b663086f931fe95